ci/test_complete.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+# Compares options in `rg --help` output to options in zsh completion function
+
+set -e
+
+main() {
+    local rg="target/${TARGET}/release/rg"
```

Comment:
Yeah, i could easily have replicated most of the find/replace stuff with `grep` and/or `sed`, but i figured that since i needed `rg` for the `--help` output anyway i might as well just use it for everything